ecs_fargate/README.md

```diff
@@ -180,6 +184,49 @@ Datadog's default CloudWatch crawler polls metrics once every 10 minutes. If you
 
 3. Finally, use a [Lambda function][22] to collect logs from CloudWatch and send them to Datadog.
 
+#### Fluent Bit and FireLens
+
+Configure the AWS FireLens integration built on Datadog's Fluent Bit output plugin to connect your FireLens monitored log data to Datadog Logs. 
+
+1. Enable the [Fluent Bit Datadog integration][27]. For more information, see the dedicated [Fluent Bit docs][27].
```

Comment:
this is actually not required because it's pre-configured by AWS and not under the control of the user. recommend to remove this step completely